liam2/data.py

```diff
@@ -712,20 +712,31 @@ def load_path_globals(globals_def):
     for name, global_def in globals_def.iteritems():
         if 'path' not in global_def:
             continue
+
         kind, info = load_def(localdir, name, global_def, [])
         if kind == 'table':
             fields, numlines, datastream, csvfile = info
             array = stream_to_array(fields, datastream, numlines)
+        elif kind == 'scalar':

```

Comment:
are these two lines of any use? I do not see anything producing that kind = "scalar"???